<html>
  <head>
    <title>BioPub</title>
    <link rel="stylesheet" href="site.css">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script async defer src="https://hypothes.is/embed.js"></script>
  </head>
  <body>
    <header class="header">
      <div class="logo">
        <img class="logo__img" src="icons/logo.svg">
        <span class="logo__bio">Bio</span>
        <span class="logo__pub">PUB</span>
      </div>
      <div class="header__divider"></div>
      <a href="#" class="header-link">Home</a>
      <a href="#" class="header-link">News</a>
      <a href="#" class="header-link">Journals</a>
      <a href="#" class="header-link">Topics</a>
      <a href="#" class="header-link">Careers</a>
    </header>

    <main class="article">
      <h1 class="article-title">Understanding the Regulation of Metabolic Enzyme Acetylation in E. Coli</h1>
      <div class="article-meta">Arti Walker-Peddakotla</div>
      <button class="cta-button" data-hypothesis-trigger>
        View annotations <span class="cta-button__count" data-hypothesis-annotation-count></span>
      </button>

      <section class="section">Abstract</section>
      <p class="article-text">Global protein acetylation is a newly discovered phenomenon in bacteria. Of the more than 250 acetylations reported in E. coli, many are of metabolic enzymes. Thus, acetylation could represent a novel posttranslational mechanism of metabolic control. Yet, almost nothing is known about the regulation of these acetylations or of their metabolic outcomes. Here, we report that the cAMP receptor protein (CRP) regulates protein acetylation in E. coli and provide evidence that protein acetylation modulates the flux of carbon through central metabolism. When we grew cells in mixed amino acids supplemented with glucose and cAMP, global protein acetylation increased in a CRP-dependent manner and several of the acetylated proteins were central metabolic enzymes. Much of this CRP-mediated acetylation required activation region 1 (AR1), a surface patch that allows CRP to interact with RNA polymerase. A second surface patch (AR2) also was involved, albeit to a lesser degree. These results raise the possibility that CRP might regulate the transcription of a protein acetyltransferase. Indeed, a recent report suggested that CRP might regulate transcription of the protein acetyltransferase YfiQ (also known as Pat) by a mechanism that would require AR2. We further obtained bioinformatic evidence that supports the hypothesis that CRP also could regulate yfiQ transcription in an AR1-dependent manner. Since CRP regulates metabolism, we asked if YfiQ could influence metabolism. Using Phenotype MicroArrayï£ª analysis, we found that a yfiQ null mutant exhibits a distinctive defect during growth on gluconeogenic
carbon sources and a distinct advantage during growth on a carbon source that
bypasses the need for gluconeogenesis. In vitro acetylation assays identified
four substrates of YfiQ. Three YfiQ substrates were the strictly irreversible
glycolytic enzymes PfkA, PfkB, and LpdA. The fourth was CRP itself. We thus
hypothesize that CRP activates yfiQ transcription, whose protein product
acetylates a subset of metabolic enzymes, altering their function and shifting
the balance between glycolysis and gluconeogenesis. We further propose that YfiQ
acetylates CRP. Efforts to determine how this acetylation affects the
CRP-dependent transcriptome are underway.</p>
    </main>

    <!--
      The footer and header currently look the same for the most part, so we just re-use the
      styles.
    !-->
    <footer class="footer">
      <a href="#" class="footer-link">About BioPub</a>
      <a href="#" class="footer-link">Jobs</a>
      <a href="#" class="footer-link">Resources</a>
      <a href="#" class="footer-link">Contact</a>
      <a href="#" class="footer-link">Terms and Policy</a>
      <span class="u-stretch"></span>
      BioPub. Subject to a Creative Commons Attribution license.
    </footer>
  </body>
</html>
